src/main/java/com/sun/caresyncsystem/aop/LoggingAspect.java

```diff
@@ -0,0 +1,21 @@
+package com.sun.caresyncsystem.aop;
+
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.*;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Aspect
+@Component
+public class LoggingAspect {
+
+    @Pointcut("within(com.sun.caresyncsystem.exception.GlobalExceptionHandler)")
```

Comment:
theo a phần Pointcut nên apply cho toàn bộ app chứ nhĩ?
```suggestion
    @Pointcut("execution(* com.sun.caresyncsystem..*.*(..))")
```